options/linux/include/sys/sysinfo.h

```diff
@@ -18,7 +20,11 @@ struct sysinfo {
 	unsigned long totalhigh;
 	unsigned long freehigh;
 	unsigned int mem_unit;
+
+	/* This is how the kernel header defines it, so suppress the warning. */
+#if !defined(__MLIBC_PEDANTIC_HEADER_CHECKER)
```

Comment:
Is the libc still conforming if this check fails?

Reply:
I couldn't come up with a better idea. AFAIK this feature shouldn't be gated behind `_GNU_SOURCE`?

Reply:
added pragma